src/AyonCppApi/AyonCppApi.cpp

```diff
@@ -116,7 +287,7 @@ AyonApi::rootReplace(const std::string &rootLessPath) {
                 return rootedPath;
             }
             catch (std::out_of_range &e) {
-                m_Log->warn("AyonApi::rootedPath error acured {}, list off available root replace str: ");
+                m_Log->warn("AyonApi::rootedPath error acured {}, list off available root replace str: ", e.what());
```

Comment:
```suggestion
                m_Log->warn("AyonApi::rootedPath error occured {}, list off available root replace str: ", e.what());
```

Reply:
The `off` here is also a typo:
```suggestion
                m_Log->warn("AyonApi::rootedPath error occured {}, list of available root replace str: ", e.what());
```